### ISSUE TITLE
As a league owner or manager, I can add a new machine to a location of play

### ISSUE THREAD

**Issue by alexstojda** [alexstojda/pinman#5]
Machines are added to a location of play using their OPDB ID. https://opdb.org/about

Blocked by #4

**Comment by alexstojda** [comment]
Closing as "won't do". We'll rely on PinballMap and it's [API](https://pinballmap.com/api/v1/docs) for locations and their machines
